### DISCUSSION TITLE
Suggestions for improving the community

### DISCUSSION THREAD

**Discussion by spacecowboy** [spacecowboy/Feeder#217] [General]
Hi everyone 

If anyone has suggestions on how to improve/grow the community around Feeder please write so here.

**Comment by Terkyz** [comment]
You can create a subreddit or Lemmy community.

**Comment by MrCamel999** [comment]
A matrix room might be a good idea

**Reply by spacecowboy** [reply]
I've been wanting to try out matrix for some time.

I've created a test space here: https://matrix.to/#/#feeder:matrix.org
